site/en/docs/android/custom-tabs/index.md

```diff
@@ -98,7 +122,7 @@ issues on [crbug.com][3] and ask questions to our Twitter account
 
 ## Getting Started
 
-If you are getting started with Custom Tabs, checkout the [Implementation Guide][11] and the 
```

Comment:
Linking to the how to section is not ideal. Can you please instead list all the sub guides.